kubeflow/asm/Makefile

```diff
@@ -33,9 +33,9 @@ download-asm-package:
 	rm -rf asm.tar.gz
 	curl -LJ https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/archive/refs/tags/$(ASM_PACKAGE_VERSION).tar.gz -o asm.tar.gz 
 	
-	rm -rf ./package
-	mkdir package
-	tar -xf asm.tar.gz --strip-components=1 -C package 
+	rm -rf $(UPSTREAM)
```

Comment:
Question: Can we use `./$(UPSTREAM)` instead of `$(UPSTREAM)`? Just want to make sure we are deleting the folder in current path.

Reply:
Changed upstream variable to `./package`. It seems to have no effect on the results of `hydrate` and `apply` running from `kubeflow` (using config.yaml or just by running `make -C asm ...`) and directly from `asm` folders. Does it address your concern?